diktat-maven-plugin/src/test/kotlin/org/cqfn/diktat/plugin/maven/DiktatMavenPluginIntegrationTest.kt

```diff
@@ -0,0 +1,35 @@
+package org.cqfn.diktat.plugin.maven
+
+import com.soebes.itf.jupiter.extension.MavenGoal
+import com.soebes.itf.jupiter.extension.MavenJupiterExtension
+import com.soebes.itf.jupiter.extension.MavenTest
+import com.soebes.itf.jupiter.maven.MavenExecutionResult
+import org.junit.jupiter.api.Assertions
+import kotlin.io.path.ExperimentalPathApi
+import kotlin.io.path.readText
+
+/**
+ * Integration tests for diktat-maven-plugin.
+ * Run against the project from diktat-examples.
+ * Note: for maven itf test name should equal example project's directory name, which we have in pom.xml.
+ */
+@OptIn(ExperimentalPathApi::class)
+@MavenJupiterExtension
+class DiktatMavenPluginIntegrationTest {
+    @MavenTest
+    @MavenGoal("diktat:check@diktat")
+    fun maven(result: MavenExecutionResult) {
+        val mavenLog = result.mavenLog.stdout.readText()
+        println(mavenLog)
```

Comment:
debug println here?
Why?

Reply:
Initially it was to have access to maven logs, but it can indeed be left for debug only and disabled by default.